src/tsickle.ts

```diff
@@ -50,8 +51,43 @@ export interface AnnotatorOptions {
 
 export enum AnnotatorFeatures {
   LowerDecorators = 1 << 0,
+  /**
+   * filter out types when expanding `export * from ...`
+   */
+  FilterTypesInExportStart = 1 << 1,
+  /**
+   * emit enums as
```

Comment:
Added all information about the flags directly to the enum members, and documented that these flags will be removed and on by default once the non transformer version is dropped.